### ISSUE TITLE
Use poster.scanner.SearchPriority regardless of mjb.internet.plugin (other than IMDb)

### ISSUE THREAD

**Issue by Omertron** [YAMJ/yamj-v2#980]
Original [issue 981](https://code.google.com/p/moviejukebox/issues/detail?id=981) created by Omertron on 2009-09-08T06:50:35.000Z:

Currently, YAMJ grabs posters from the plugin site. For example, if mjb.internet.plugin=com.moviejukebox.plugin.AllocinePlugin, then YAMJ will 
be forced to get posters from http://www.allocine.fr/ regardless of how poster.scanner.SearchPriority is set up.

poster.scanner.SearchPriority seems only to work for IMDb, where it's 
assumed the language will be English. All the possible poster.scanner.SearchPriority values are geared towards English posters.

Unfortunately, sometimes the posters from a foreign plugin is rather small. 
MovieMeter uses 200x295 posters. It's unfair that MovieMeter users have to 
suffer with small posters.

It's better to use a high resolution English poster than a low resolution 
foreign one.

So basically, regardless of what mjb.internet.plugin is set to, let YAMJ 
follow poster.scanner.SearchPriority.
